### DISCUSSION TITLE
Code generated schema linting?

### DISCUSSION THREAD

**Discussion by neviaumi** [graphql-hive/graphql-eslint#994] [Q&A] [answered]
I currently usage NestJS as backend and I want to enable some rules that need schema and operation has been provided like require-id rules 

What should I do?

Ref: https://docs.nestjs.com/graphql/quick-start

Given documentation, I guess I can enable auto generation schema with file path and sort schema options in order to get thing work but do I have better options?

**Comment by dimaMachina** [comment, marked as answer]
Hi @davidNHK, sorry for the long response, you can provide a path to your generated SDL schema (if you generate it) or a URL endpoint of your schema.

**Reply by neviaumi** [reply]
Hey! No worries 😉 turn out My solution here let nest generate the SDL schema and commit to git 

so I can use it when CI
